### PR TITLE
Add Pireps and Fix METAR Altitude Issue.

### PR DESCRIPTION
**Pull request by ironpilot**

### CODE REVIEW

File: src/components/pirep-list-item/pirep-list-item.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+						<span *ngIf="icing.top">{{ icing.top }} (Top)</span>
+					</div>
+				</ion-col>
+			</ion-row>
+		</ion-col>
+	</ion-row>
+	<ion-row *ngIf="pirep.turbulenceCondition.length > 0">
+		<ion-col col-12>
+			<h2>Turbulence</h2>
+			<ion-row *ngFor="let turbulence of pirep.turbulenceCondition; let i = index">
```
Comment (AlexAlappsis) [inline src/components/pirep-list-item/pirep-list-item.html]: Not using i?

File: src/components/pirep-list-item/pirep-list-item.html [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+						<span *ngIf="turbulence.top">{{ turbulence.top }} (Top)</span>
+					</div>
+				</ion-col>
+			</ion-row>
+		</ion-col>
+	</ion-row>
+	<ion-row *ngIf="pirep.skyCondition.length > 0">
+		<ion-col col-12>
+			<h2>Sky Conditions</h2>
+			<ion-row *ngFor="let condition of pirep.skyCondition; let i = index">
```
Comment (AlexAlappsis) [inline src/components/pirep-list-item/pirep-list-item.html]: i?

File: src/pages/flight-plan/flight-plan.html [hunk 3]
```diff
@@ -103,6 +103,21 @@ <h2>
...
+		<ion-list-header>
+			<button ion-item (click)="onToggleSection(sections.airsigmets)" detail-none
+					[ngClass]="{'section-active': sections.airsigmets.open, 'section': !sections.airsigmets.open}">
+				<ion-icon item-left name="arrow-forward" *ngIf="!sections.airsigmets.open"></ion-icon>
+				<ion-icon item-left name="arrow-down" *ngIf="sections.airsigmets.open"></ion-icon>
+				{{ sections.airsigmets.title }}
+			</button>
+
+			<ion-list *ngIf="sections.airsigmets.open">
+				<ion-item *ngFor="let met of airsigmets">
```
Comment (AlexAlappsis) [inline src/pages/flight-plan/flight-plan.html:116]: Empty item.

Comment (ironpilot) [reply src/pages/flight-plan/flight-plan.html:116]: You are correct this is empty currently! This is only a placeholder for when I get the AIRMET information filled in here.

File: src/components/pirep-list-item/pirep-list-item.html (hunk 1 above)
Comment (ironpilot) [reply src/components/pirep-list-item/pirep-list-item.html]: Good catch. Left over from copied code. I will fix.

File: src/components/pirep-list-item/pirep-list-item.html (hunk 2 above)
Comment (ironpilot) [reply src/components/pirep-list-item/pirep-list-item.html]: Fixed

**Pull request merged**